diff --git a/app/Kconfig b/app/Kconfig
index 8c0675e0..c6c4ed94 100644
--- a/app/Kconfig
+++ b/app/Kconfig
@@ -663,13 +663,6 @@ endmenu # Advanced
 
 endmenu # ZMK
 
-if SOC_FAMILY_NORDIC_NRF
-
-config NRF_SOC_VALIDATE_HEADERS_DISABLED
-    default y
-
-endif
-
 config KERNEL_BIN_NAME
     default "zmk"
 
diff --git a/app/boards/shields/nice_view/Kconfig.defconfig b/app/boards/shields/nice_view/Kconfig.defconfig
index 0b2b7be3..cafeaa47 100644
--- a/app/boards/shields/nice_view/Kconfig.defconfig
+++ b/app/boards/shields/nice_view/Kconfig.defconfig
@@ -3,9 +3,6 @@
 
 if SHIELD_NICE_VIEW
 
-config LS0XX_VCOM_THREAD_PRIO
-    default 11
-
 config LV_Z_VDB_SIZE
     default 100
 
